packages/mdl-card/mdl-card.scss

```diff
@@ -105,6 +106,22 @@
     box-sizing: border-box;
   }
 
+  &__media-item {
+    display: inline-block;
+    width: auto;
+    height: 80px;
+    margin: 16px 0 0;
+    padding: 0;
+
+    @each $mult, $name in (1: "1", 1.5: "1dot5", 2: "2", 3: "3") {
```

Comment:
It seems to me like `1` here is the default, so what scenario necessitates including a modifier for it?